net/dnscrypt-proxy2/Makefile

```diff
@@ -22,7 +22,7 @@ PKG_LICENSE_FILES:=LICENSE
 
 PKG_BUILD_DEPENDS:=golang/host
 PKG_BUILD_PARALLEL:=1
-PKG_USE_MIPS16:=0
```

Comment:
Keep it as-is, build flags is only available on snapshot right now

Reply:
OK, I will remove it